install_bootstrap

```diff
@@ -323,6 +326,8 @@ while [[ "${run_boardwalk^^}" != 'Y' &&  "${run_boardwalk^^}" != 'N' ]] ; do
         azul_s3_bucket='azul_s3_bucket'
         ask_question "What is the name of the AWS S3 bucket to store BDBags?" "$AZUL_S3_BUCKET" "Azul S3 Bucket" $azul_s3_bucket
 
+	azul_presigned_url_expiration='azul_presigned_url_expiration'
+	ask_question "Choose an expiration time (in seconds) for presigned URL to the S3-BDBag for FC export (recommended 3600)?" "$AZUL_PRESIGNED_URL_EXPIRATION" "Azul Presigned Url Expiration" $azul_presigned_url_expiration
```

Comment:
If you keep this, either `What is...?`, or `Choose...` with no question mark at the end. But I don't think you need this question...  I think you should hard-code the value like `es_port` and `es_protocol` are hard-coded with the sensible default you have in your code.

My point is that a user will typically not need to change this, just like they don't need to change the `es_port`. But on the off chance that they do, we will have an environment variable than can be modified without changing the code.

Reply:
I followed your suggestion, and removed the question.